rust/htp/Cargo.toml.in

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "suricata-htp"
-version = "2.0.0"
```

Comment:
Does this mean that htp version will be updated with every Suricata version irrespective of whether there was a change in libhtp?

Reply:
Yes, it does.